test_vm/src/lib.rs

```diff
@@ -107,6 +107,21 @@ pub trait VM<BS: Blockstore> {
 
     /// Take all the invocations that have been made since the last call to this method
     fn take_invocations(&self) -> Vec<InvocationTrace>;
+
+    /// Get information about an actor
+    fn actor(&self, address: &Address) -> Option<Actor>;
+
+    /// Build a map of all actors in the system and their type
```

Comment:
"... of all actor code types in the system..." I think is more accurate. The map key looks like a code CID rather than address.